src/app/containers/Blocks/index.jsx

```diff
@@ -8,14 +8,23 @@ const BlockString = props => {
   return <p>{stringProps}</p>;
 };
 
-const Blocks = ({ blocks, componentsToRender }) =>
+const Blocks = ({ blocks, componentsToRender, isTopLevel }) =>
   blocks.map((block, index) => {
     const { type, model } = block;
 
     const { type: typeOfPreviousBlock } = blocks[index - 1] || {};
 
     const Block = componentsToRender[type] || BlockString;
-
+    if (isTopLevel) {
```

Comment:
This logic isn't doing anything different in this PR, can it be removed?

Reply:
Agreed, let's not abstract too early!